config/config.toml.example

```diff
@@ -0,0 +1,167 @@
+# A sample configuration for fedora-messaging. This file is in the TOML format.
+# For complete details on all configuration options, see the documentation
+# https://fedora-messaging.readthedocs.io/en/latest/configuration.html.
+
+amqp_url = "amqp://"
+
+publish_exchange = "amq.topic"
+
+callback = "hotness.consumers:BugzillaTicketFiler"
+
+# Note the double brackets below.
+# To add another binding, add another [[bindings]] section.
+[[bindings]]
+queue = "anitya"
+exchange = "amq.topic"
+routing_keys = [
+    "org.release-monitoring.prod.anitya.project.version.update",
+    "org.release-monitoring.prod.anitya.project.map.new",
+]
+
+[[bindings]]
+queue = "koji"
+exchange = "amq.topic"
+routing_keys = [
+    "org.fedoraproject.prod.buildsys.task.state.change",
+]
+
+[tls]
+ca_cert = "/etc/pki/tls/certs/ca-bundle.crt"
+keyfile = "/my/client/key.pem"
+certfile = "/my/client/cert.pem"
+
+[client_properties]
+app = "the-new-hotness"
+
+[exchanges.custom_exchange]
+type = "fanout"
+durable = false
+auto_delete = false
+arguments = {}
+
+[queues.anitya]
+durable = true
+auto_delete = false
+exclusive = false
+arguments = {}
+
+[queues.koji]
+durable = true
+auto_delete = false
+exclusive = false
+arguments = {}
+
+[qos]
+prefetch_size = 0
+prefetch_count = 25
+
+[log_config]
+version = 1
+disable_existing_loggers = true
+
+[log_config.formatters.simple]
+format = "[%(name)s %(levelname)s] %(message)s"
+
+[log_config.handlers.console]
+class = "logging.StreamHandler"
+formatter = "simple"
+stream = "ext://sys.stdout"
+
+[log_config.loggers.fedora_messaging]
+level = "INFO"
+propagate = false
+handlers = ["console"]
+
+[log_config.root]
+level = "WARNING"
+handlers = ["console"]
+
+# the-new-hotness consumer configuration
+[consumer_config]
+pkgdb_url = "https://admin.fedoraproject.org/pkgdb/api"
+mdapi_url = "https://apps.fedoraproject.org/mdapi"
+# The time in seconds the-new-hotness should wait for a socket to connect
+# before giving up.
+connect_timeout = 15
+# The time in seconds the-new-hotness should wait for a read from a socket
+# before giving up.
+read_timeout = 15
+# The number of times the-new-hotness should retry a network request that
+# that failed for any reason (e.g. read timeout, DNS error, etc)
+requests_retries = 3
+# If true, publish fedmsg messages instead of fedora-messaging messages
+legacy_messaging = false
```

Comment:
Hmm OK but you're still only *consuming* from fedora-messaging, and not from fedmsg, right?

Reply:
Yes, this is only for publishing.